packages/select/src/Select/index.ts

```diff
@@ -1,6 +1,8 @@
 export { Select } from './Select';
```

Comment:
Not sure what's causing this (might be related to what we chatted about a few weeks ago), but there seems to be a slight delay before the menu opens https://jmp.sh/XXm7a3hF